ionoscloud/data_source_dbaas_mongo_template.go

```diff
@@ -94,6 +97,10 @@ func dataSourceDbassMongoTemplateRead(ctx context.Context, d *schema.ResourceDat
 	} else if len(templates) > 1 {
 		return diag.FromErr(fmt.Errorf("more than one DBaaS Mongo Template found for the specified search criteria"))
 	}
+
+	if err := d.Set("id", *templates[0].Id); err != nil {
```

Comment:
@cristiGuranIonos I see that in `SetMongoDBTemplateData()` call from below we are setting the ID, why are we adding this other set here?

Reply:
This is not something that I could reproduce. So I'm covering both bases, making is computed and setting it explicitly

Reply:
@cristiGuranIonos I am not sure that this will solve the problem but we will see.